pkg/controller/networkpolicy/validate.go

```diff
@@ -409,6 +411,10 @@ func (t *tierValidator) updateValidate(curObj, oldObj interface{}, userInfo auth
 	reason := ""
 	curTier := curObj.(*secv1alpha1.Tier)
 	oldTier := oldObj.(*secv1alpha1.Tier)
+	// Allow exception of Tier Priority updates performed by the antrea-controller
+	if serviceaccount.MatchesUsername("kube-system", env.GetAntreaControllerServiceAccount(), userInfo.Username) {
```

Comment:
"kube-system" should come from `POD_NAMESPACE` env variable, with "kube-system" used as the default